### DISCUSSION TITLE
Odd Travel Moves in SS

### DISCUSSION THREAD

**Discussion by djngnear** [supermerill/SuperSlicer#2109] [General]
I've noticed from SS 2.3.56 (I think it was) that the nozzle will take slight detours instead of going straight to where it needs to go. These pictures are of the same file sliced in SS and PS. You can see the nozzle does slight moves left or right before ending up at the next extrusion. I know it's not the biggest issue but less travel time would mean less oozing filament. And I just sliced it in the latest 2.3.57.8 and there's no change. Maybe someone knows what's going on here? Thanks.
 
![SS Travel Moves](https://user-images.githubusercontent.com/87901234/147164538-c76b978e-1197-48c8-88c7-4d793d020d9e.JPG)

![PS Travel Moves](https://user-images.githubusercontent.com/87901234/147164549-81d45409-6433-4692-b82d-dc1294f14701.JPG)

**Comment by supermerill** [comment]
You should have oppened an issue.
Can you provide your project file?

**Comment by djngnear** [comment]
Yes sorry I haven't been on here much.

https://drive.google.com/file/d/1_ymHdMlFttOvfKIZX8Q79fExRPvM8fAc/view?usp=sharing
